.ci/generate-extra-site-links.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -e
+
+source ./.ci/util.sh
+
+if [[ -z $1 || -z $2 ]]; then
```

Comment:
```suggestion

PR_NUMBER=$1
AWS_FOLDER_LINK=$2

if [[ -z $PR_NUMBER || -z $AWS_FOLDER_LINK ]]; then
```
Why do we need to be cryptic in checking for empty string/null?

Reply:
Done.